index.html

```diff
@@ -7,7 +7,96 @@
   <title>Document</title>
 </head>
 <body>
-  <h1>Hello World!</h1>
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/igVzAG-fih8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    <table>
```

Comment:
Use of tables for layout is deprecated. It was widely used before CSS, but all visual presentation is now written with CSS. Instead, use HTML elements that provide meaningful structure. It will not look like your wireframe at this point and that is okay. 

I recommend the use of `<header>`, `<main>`, and `<footer>` elements as a first step in creating a meaningful structure for your pages.